addons/web/static/src/js/services/crash_manager.js

```diff
@@ -129,24 +130,47 @@ var CrashManager = core.Class.extend({
             this.show_error(error);
         }
     },
+    getMetaBtns: function (metadata) {
```

Comment:
I agree with you on the metadata not meaning anything, it was originally called a 'post-mortem action' but @antonylesuisse insisted on it being called something like 'metadata' because I think he wants it to be generic for future uses...

Mind you this is not for server actions only, so I would call it simply 'action'

Reply:
`actions` is nice :)
if in the future it needs to be extended to more stuff, we will call it something less specific (like `stuff`)